diff --git a/src/pysqa/wrapper/flux.py b/src/pysqa/wrapper/flux.py
index 5ad3b8cf..1a684c7c 100644
--- a/src/pysqa/wrapper/flux.py
+++ b/src/pysqa/wrapper/flux.py
@@ -17,8 +17,8 @@
 {%- if run_time_max %}
 # flux: -t {{ [1, run_time_max // 60]|max }}
 {%- endif %}
-{%- if dependency %}
-{%- for jobid in dependency %}
+{%- if dependency_list %}
+{%- for jobid in dependency_list %}
 # flux: --dependency=afterok:{{jobid}}
 {%- endfor %}
 {%- endif %}
diff --git a/src/pysqa/wrapper/slurm.py b/src/pysqa/wrapper/slurm.py
index 7f1a1991..dd06882e 100644
--- a/src/pysqa/wrapper/slurm.py
+++ b/src/pysqa/wrapper/slurm.py
@@ -16,8 +16,8 @@
 {%- if run_time_max %}
 #SBATCH --time={{ [1, run_time_max // 60]|max }}
 {%- endif %}
-{%- if dependency %}
-#SBATCH --dependency=afterok:{{ dependency | join(',') }}
+{%- if dependency_list %}
+#SBATCH --dependency_list=afterok:{{ dependency_list | join(',') }}
 {%- endif %}
 {%- if memory_max %}
 #SBATCH --mem={{memory_max}}G
diff --git a/src/pysqa/wrapper/torque.py b/src/pysqa/wrapper/torque.py
index ef4c16b1..2b8b9bce 100644
--- a/src/pysqa/wrapper/torque.py
+++ b/src/pysqa/wrapper/torque.py
@@ -18,8 +18,8 @@
 #PBS -l walltime={{run_time_max}} 
 {%- endif %}
 #PBS -l wd
-{%- if dependency %}
-#PBS -W depend=afterok:{{ dependency | join(':') }}
+{%- if dependency_list %}
+#PBS -W depend=afterok:{{ dependency_list | join(':') }}
 {%- endif %}
  
 {{command}}
